locales/mr-IN.ftl

```diff
@@ -0,0 +1,38 @@
+Title = Firefox Quantum रिलीझ मोहीम
+
+Project-name = Firefox Quantum रिलीझ
+
+main-title = Firefox परत आले आहे!
+
+main-description = आम्हाला नवीन Firefox बद्दल जगात प्रसारण करण्यास मदत करा! मंगळवार 14 ला,14:00 UTC वाजता #NewFirefox वापरून आपल्याला त्यातील काय आवडते त्या विषयी ट्विट करा किंवा
+Firefox चे ट्विट परत रिट्विट करा.
+
+thunderclap-cta = Firefox Quantum चे समर्थन करा
+
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+
+official-tweet-cta = अधिकृत ट्विट रिट्विट करा
+
+Tweet-1 = आज #NewFirefox आले आहे आणि तेही पूर्वीपेक्षा दुप्पट जलद :boom: डाउनलोड करा:https://mzl.la/newfirefox
```

Comment:
please notice the key, it should be "tweet-1", lowercase. Also, the :boom: won't work, please copy the emoticon 1:1 from the english file (and if possible not from github, that leads to the :boom: instead of it being a real emoticon).